ArduCopter/mode_autotune.cpp

```diff
@@ -38,30 +38,20 @@ void AutoTune::run()
     // apply SIMPLE mode transform to pilot inputs
     copter.update_simple_mode();
 
-    // reset target lean angles and heading while landed
+    // disarm when the landing detector says we've landed and spool state is ground idle
```

Comment:
disarming when the vehicle becomes landed is available as a flight option but its not the default so I think this would surprise users

Reply:
Well, we would never allow the user to takeoff in autotune so why would we leave the aircraft in an armed state after landing.  Plus disarming would then save the gains which i think is helpful for the user.

Reply:
I think we don't allow users to arm in autotune but they can takeoff in autotune.  It's really more a matter of consistency though.  If we want the vehicle to always disarm while landed we can discuss and perhaps make that change but having different modes behave differently is confusing to the user

Reply:
There isn’t any logic in autotune to support takeoff.

Reply:
I tend to agree with @rmackay9 I want disarm to be under my control, not autotune - this would also be a surprise.

Reply:
The alternative is to ignore any positive climb rate or switch out of mot_spin_arm after landed but still let the user disarm.